packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -304,13 +304,11 @@ export interface SchemaStatics {
 }
 
 /**
- * Stateless APIs exposed via {@link SchemaFactory} as both instance properties and as statics.
- * @privateRemarks
- * We have no way to make linkable members which exist both as statics and instance properties since API-Extractor does not support this.
- * As a workaround, we have this type as a third place which can be linked.
- * @system @sealed @alpha
+ * Unstable extensions to SchemaStaticsAlpha.
+ * @remarks
+ * Only items missing in schemaStatics are added here: items which simply have more specific types in alpha are instead just included in {@link schemaStatics} directly.
```

Comment:
This makes me think `SchemaStaticsAlpha` may not be the right name.

Reply:
fixed

Reply:
I fixed the incorrect mention to SchemaStaticsAlpha in the description. I guess you mean't something else?